packages/lib/src/core/ProcessResponse/PaymentAction/actionTypes.ts

```diff
@@ -31,7 +31,7 @@ const actionTypes = {
             paymentData: action.paymentData,
             onComplete: props.onAdditionalDetails,
             onError: props.onError,
-            size: '05',
+            size: '02',
```

Comment:
Just to make sure, can this be overriden via props?

Reply:
No. Just the way it can't be at present, either (unless you instantiate the component directly with `checkout.create('threeDS2Challenge')`)

Reply:
Would it be possible to allow overriding this value also when not using the new flow?

Reply:
Why suddenly introduce that option when its _never_ been there before?
Why not just move to a new version if you want this config option?